packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -305,6 +306,7 @@ export const VoiceProvider: FC<VoiceProviderProps> = ({
     try {
       await client.connect({
         ...config,
+        verboseTranscription: true,
```

Comment:
Is there a reason we need to explicitly set it here if it defaults to true?

Reply:
it doesn't default to true on the backend, but we want it to default to true on the react SDK to enable more sensitive interruptions